download.py

```diff
@@ -42,7 +42,7 @@ def download_run_fasterq_dump(accession, term, total_spots, out):
     )
     if correctness:
         logging.info("")
```

Comment:
Maybe we need to add a logging message or remove the line completely?
I would suggest the first option - to add a message like "start compressing the %filename%..." or something like that.

Reply:
| FWIW I was going to change that in my next pull request

Nobody said you must do it right here - or die  :) It was a suggestion, you are free to disagree and provide the reason why you disagree

|  in general I would prefer it if reviews were about modified code.

Yeah, I agree with you on that. However, such small things can be easily forgotten if we don't fix them right away. And it's not as if we are modifying the logic here that we are not supposed to modify within the given task. Just a tiny little fix of log message - not a big deal, imo